### ISSUE TITLE
Large biome worlds displaying as default

### ISSUE THREAD

**Issue by hjpoe** [toolbox4minecraft/amidst#800]
Hello,

I don't know if this is a known error or not, and I didn't see it in the issues, so I thought I'd post it. 

One of the worlds I'm playing should look like this (I think I made it large biome, but I'm never 100% sure). 

![image](https://user-images.githubusercontent.com/35583004/86304667-97f7fe00-bbdd-11ea-824a-fe53d6ccbca0.png)

I'm currently at that village on the island at ~2100 ~40. When I open the world save in amidst, it shows me this map with me in the middle of the ocean. 

![image](https://user-images.githubusercontent.com/35583004/86305238-32a50c80-bbdf-11ea-8957-020d235583c3.png)

**Comment by machinesmith42** [comment]
currently out of scope?

**Comment by hjpoe** [comment]
It doesn't seem like it should be out of scope. Wouldn't you just have to
make the existing map bigger? I could help if I knew what to do, I have
experience in Java.

**Comment by burgerguy** [comment]
Large biomes is definitely within the scope of amidst considering that deliberate decisions were made in the code to allow it. I’ll take a look at it later when I get the chance to see what’s wrong with it.

**Comment by hjpoe** [comment]
Thanks, I thought that they would still be supported. If there's anything I
can do to help, please let me know. I'm not very familiar with your code
because I haven't read through it yet, but I can program.

**Comment by burgerguy** [comment]
Here are two screenshots with the same zoom and the same seed, one set to large biomes:

Default:
![screenshot_default_-703112092150021424](https://user-images.githubusercontent.com/30326913/86391919-8f003e80-bc68-11ea-9147-85948e8910af.png)

Large Biomes:
![screenshot_large-biomes_-703112092150021424](https://user-images.githubusercontent.com/30326913/86391924-91629880-bc68-11ea-839e-5f894f6820a4.png)

Are you sure that you enabled large biomes when typing in the seed? It seems to work fine on the latest version of amidst.

------------
Also, if you want to help out with programming you can read the wiki for how to get started. As for becoming familiar with the amidst codebase, you really just need to read through it all and start working with it. Knowing how everything interacts with each other comes over time. I'm not really the best person to ask for what you can do at the moment, but you could ask @moulins and he would be able to help.

**Comment by hjpoe** [comment]
I know Amidst can show a large biome world from the seed. My problem is with opening a save game.

My first screenshot was opened new from seed. The issue is when I opened the world via open save game, it shows it as default biome size, when I am clearly not in the ocean, as I (was) at a village. I'm not 100% positive that I selected large biome size, but evidence seems to point that way. I have the latest version of Amidst. 

Thanks for the information, I'll look at the wiki.
